src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -740,10 +740,10 @@ window.qBittorrent.DynamicTable = (function() {
                     const tr = new Element("tr");
                     // set tabindex so element receives keydown events
                     // more info: https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event
-                    tr.setProperty("tabindex", "-1");
+                    tr.tabindex = "-1";
```

Comment:
This change broke #15186. It should be a real attribute to work.

Reply:
It's more than that actually. https://github.com/qbittorrent/qBittorrent/pull/21147#discussion_r1807361385
I would address it in a new PR.